### PR TITLE
Add example that retrieves image from a camera

### PR DESCRIPTION
**Pull request by daoo**
Add example that retrieves a single image frame from a single camera.

Example invocation:
```
$ python examples/image_example.py -o C:/temp/image.jpg -c 1 -p password
2024-06-07 13:31:52,685 - qtm_rt - INFO - Connected
2024-06-07 13:31:52,686 - root - INFO - Setting Enabled for Camera 1 to True
2024-06-07 13:31:52,687 - qtm_rt - INFO - Took control
2024-06-07 13:31:52,705 - root - INFO - RTImageComponent(image_count=1)
2024-06-07 13:31:52,705 - root - INFO - RTImage(id=1, format=2, width=736, height=720, left_crop=0.11666666716337204, top_crop=0.0, right_crop=0.8833333253860474, bottom_crop=1.0, image_size=122882)
2024-06-07 13:31:52,706 - root - INFO - Writing C:/temp/image.jpg
2024-06-07 13:31:52,706 - qtm_rt - INFO - Released control
2024-06-07 13:31:52,706 - qtm_rt - INFO - Disconnected
```

### CODE REVIEW

File: examples/image_example.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    format = output_format(output_path)
+    xml = ET.fromstring(xml_str)
+    for camera in xml.findall("./Image/Camera"):
+        camera_id = camera.find("ID").text
+        is_target_camera = camera_id == target_camera_id
+        logging.info("Setting Enabled for Camera %s to %s", camera_id, is_target_camera)
+        camera.find("Enabled").text = str(is_target_camera)
+
+        if is_target_camera:
+            camera.find("Format").text = format
```
Comment (OliverGlandberger) [inline examples/image_example.py:33]: Some might consider this a stupid question, but I want to ask it regardless: **Can more than one camera have the same ID?** Purely based on what "ID" means in my world view, I would assume each camera has a unique ID. If this is the case, do we want to exit the for-loop early? To make the code a little more explicit?

File: examples/image_example.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+    xml.tag = "QTM_Settings"
+    return ET.tostring(xml).decode("utf-8")
+
+
+async def main(password, target_camera_id, output_path):
+    connection = await qtm_rt.connect("127.0.0.1")
+
+    if connection is None:
+        raise RuntimeError("Failed to connect")
```
Comment (OliverGlandberger) [inline examples/image_example.py:43]: Is it helpful at all to provide the **target_camera_id** or **output_path** here in the RunetimeError message? If the likelihood of it being helpful is 0% feel free to just ignore this comment, else consider adding one or the other, or both.

**Review by OliverGlandberger — CHANGES_REQUESTED** [review]
Looks good to me! Resolve the comments I left as you see fit. 👍

**Review by bahlborn — APPROVED** [review]

File: examples/image_example.py (hunk 2 above)
Comment (daoo) [reply examples/image_example.py:43]: Good idea in general to include more info in error messages. Though, in this case, the failure to connect have nothing to do with the `target_camera_id` or the `output_path`. So including them is not useful.

File: examples/image_example.py (hunk 1 above)
Comment (daoo) [reply examples/image_example.py:33]: No, the IDs are unique. They are basically the camera numbers as QTM counts/numbers the cameras.

**Pull request merged**